### PR TITLE
Autocompletes appearance, login alert, menu icons and show resolution parameters

### PR DESCRIPTION
**Pull request by dlopbar**
* Show resolution parameters
* Changes in autocompletes appearance
* Login alert
* Menu icons 
* Links to baobab webs
* (don't know if changes in package.json and url are necessary)

### CODE REVIEW

File: src/store.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ export default new Vuex.Store({
     barColor: 'rgba(0, 0, 0, .8), rgba(0, 0, 0, .8)',
     barImage: 'https://demos.creative-tim.com/material-dashboard/assets/img/sidebar-1.jpg',
     drawer: null,
-    url: 'http://127.0.0.1:5000',
+    url: 'http://34.76.18.17:5000',
```
Comment (pchtsp) [inline src/store.js]: this change is not necessary. You can have the test url but don't commit it into github.

File: src/views/dashboard/pages/UserProfile.vue [hunk 2]
```diff
@@ -125,6 +140,16 @@
...
+    props: {
+      submitted: {
+        type: Boolean,
+        default: false,
+      },
+      error: {
+        type: Boolean,
+        default: false,
+      },
+    },
```
Comment (pchtsp) [inline src/views/dashboard/pages/UserProfile.vue]: the way the alerts are shown will be more in the style of :

https://github.com/baobabsoluciones/cornflow-admin/blob/93b198d3b851595ee35cebf5606726ded0ce4695/src/views/dashboard/main/InstanceMenu.vue#L116-L119
and
https://github.com/baobabsoluciones/cornflow-admin/blob/93b198d3b851595ee35cebf5606726ded0ce4695/src/views/dashboard/main/InstanceMenu.vue#L136-L145

File: src/views/dashboard/solution/ShowSolution.vue [hunk 3]
```diff
@@ -4,39 +4,116 @@
...
+        />
+      </v-col>
+    </v-row>
     <!-- <p>Original message: "{{ selectedLog }}"</p> -->
-    <progress-line-chart
+    <!-- <progress-line-chart
       :options="{ width: 800, height: 180 }"
       :dataset="dataset"
-    ></progress-line-chart>
+    ></progress-line-chart> -->
```
Comment (pchtsp) [inline src/views/dashboard/solution/ShowSolution.vue]: maybe delete the commented lines?

File: src/views/dashboard/solution/ShowSolution.vue [hunk 4]
```diff
@@ -4,39 +4,116 @@
...
+      },
+      gap: {
+        type: Number,
+        default: 0,
+      },
+      time: {
+        type: Number,
+        default: 0,
+      },
     },
```
Comment (pchtsp) [inline src/views/dashboard/solution/ShowSolution.vue]: From what I see you're using these parameters to show them inside the app and you're filling them inside the component. if you're going to modify these parameters inside this component, you need to put them inside `data`, not in `props`. `props` is for "input arguments to the component" that are passed through the template in html. An example of `props`: 

https://github.com/baobabsoluciones/cornflow-admin/blob/93b198d3b851595ee35cebf5606726ded0ce4695/src/views/dashboard/main/ExecutionsTable.vue#L64-L71

that are filled here (in another file):

https://github.com/baobabsoluciones/cornflow-admin/blob/93b198d3b851595ee35cebf5606726ded0ce4695/src/views/dashboard/main/InstanceMenu.vue#L77-L81

**Review by pchtsp — CHANGES_REQUESTED** [review]

File: src/views/dashboard/solution/ShowSolution.vue (hunk 3 above)
Comment (dlopbar) [reply src/views/dashboard/solution/ShowSolution.vue]: These commented lines only avoid the problem caused by small instances. Later they will be needed to display the graph.

**Review by pchtsp — APPROVED** [review]

**Pull request merged**